tool.ts

```diff
@@ -521,6 +523,7 @@ function _createExtractFolder(dest?: string): string {
  * @param regex     regex to use for version matches
  */
 export async function scrape(url: string, regex: RegExp): Promise<string[]> {
+    let http: httpm.HttpClient = new httpm.HttpClient(userAgent, null, requestOptions);
```

Comment:
Any reason not to allow auth handlers here as well?

Reply:
Not quite, wasn't sure if it is a desired functionality. Added the option now